internal/util-logging/src/main/scala/sbt/internal/util/ProgressState.scala

```diff
@@ -100,7 +100,10 @@ private[sbt] final class ProgressState(
           toWrite ++= (DeleteLine + ClearScreenAfterCursor + CursorLeft1000).getBytes("UTF-8")
         case _ =>
       }
-      toWrite ++= bytes
+      bytes.foreach { b =>
+        if (b == 10) toWrite ++= ClearScreenAfterCursor.getBytes("UTF-8")
```

Comment:
`0x0A` upper half of the codepage is always ASCII in UTF-8, it seems to this should be ok.

Reply:
Yeah, and the line separator is technically "\r\n" not just "\n" but for these purposes, "\n" still works for this purpose of making sure that we clear everything to the right and below the cursor.